projects/nexton/index.js

```diff
@@ -0,0 +1,35 @@
+
+const { call, getTokenRates } = require("../helper/chain/ton");
+
+const contractAddress = "EQBED_4VLQC-SRdrVCtmcmwO7PtnvdLSRFRYNaGIZBqpaQHQ";
+const nxton = {
+    token: "0:9d123d5d121efaf9069c737f19447d9e1adab2a7ee747cedffd3004a17c1ca4f",
+    name: "NxTON",
+    decimals: 9,
+};
+
+const getTVL = async (api)=> {
+    const tonStaked = (await call({
+        target: contractAddress,
```

Comment:
why not check the amount of ton tokens on the contract, it should give same value right?

Reply:
We store TON tokens in a separate wallet and run an arbitrage bot from there.